studies/serializers.py

```diff
@@ -98,3 +98,18 @@ class Meta:
             'study',
             'demographic_snapshot',
         )
+
+
+class ResponseWriteableSerializer(ResponseSerializer):
+    def create(self, validated_data):
+        """
+        Use the ids for objects so django rest framework doesn't
+        try to create new objects out of spite
+        """
+        study = validated_data.pop('study')
+        validated_data['study_id'] = study.id
+        # implicitly set the demographic data because we know what it will be
+        validated_data['demographic_snapshot_id'] = validated_data.get('child').user.latest_demographics.id
```

Comment:
`latest_demographics` (outside of this PR) might be pulling the first demographics info instead of the last?